openslides/users/static/js/users/pdf.js

```diff
@@ -213,11 +213,11 @@ angular.module('OpenSlidesApp.users.pdf', ['OpenSlidesApp.core.pdf'])
             var createWelcomeText = function() {
                 return [
                     {
-                        text:   getConfigValue('users_pdf_welcometitle'),
+                        text:   gettextCatalog.getString(getConfigValue('users_pdf_welcometitle')),
```

Comment:
Do you want to remove the getConfigValue function? Otherwise I would change this line to:
`Config.translate(getConfigValue('users_pdf_welcometitle'))`